bundle/libraries/local_path_test.go

```diff
@@ -30,24 +30,13 @@ func TestIsLocalPath(t *testing.T) {
 	assert.False(t, IsLocalPath("abfss://path/to/package"))
 }
 
-func TestIsLocalLibrary(t *testing.T) {
-	// Local paths.
-	assert.True(t, IsLocalLibrary(&compute.Library{Whl: "./file.whl"}))
-	assert.True(t, IsLocalLibrary(&compute.Library{Jar: "../target/some.jar"}))
-
-	// Non-local paths.
-	assert.False(t, IsLocalLibrary(&compute.Library{Whl: "/Workspace/path/to/file.whl"}))
-	assert.False(t, IsLocalLibrary(&compute.Library{Jar: "s3:/bucket/path/some.jar"}))
-
-	// Empty.
-	assert.False(t, IsLocalLibrary(&compute.Library{}))
-}
-
-func TestIsEnvironmentDependencyLocal(t *testing.T) {
+func TestIsLibraryLocal(t *testing.T) {
 	testCases := [](struct {
 		path     string
 		expected bool
 	}){
+		{path: "local/*.whl", expected: true},
+		{path: "local/test.whl", expected: true},
```

Comment:
Since this supersedes the test that was removed, this should also include test cases for `s3:` and `file:` prefixes.